3d_armor/init.lua

```diff
@@ -62,6 +64,13 @@ for material, _ in pairs(armor.materials) do
 	end
 end
 
+-- Convert set_elements to a Lua table splitting on blank spaces
+local t_set_elements = armor.config.set_elements
+armor.config.set_elements = {}
```

Comment:
`armor.config.set_elements = string.split(t_set_elements, " ")`

https://github.com/minetest/minetest/blob/master/builtin/common/misc_helpers.lua#L171

Reply:
Updated and retested as okay

```
-- Convert set_elements to a Lua table splitting on blank spaces
local t_set_elements = armor.config.set_elements
armor.config.set_elements = {}
armor.config.set_elements = string.split(t_set_elements, " ")
```

Reply:
`armor.config.set_elements = {}` is no longer needed if you assign the field to `string.split`, which returns a new table on its own. If you want, you could even shorten it as follows because the assignment happens after the function returned:
```Lua
armor.config.set_elements = string.split(armor.config.set_elements, " ")
```